### DISCUSSION TITLE
Razer Chroma support

### DISCUSSION THREAD

**Discussion by mortismint** [open-goal/jak-project#3078] [Ideas]
Since the LEDs for DS4's work, it would also be cool to have it working for Razer Chroma, maybe even for ICUE from Corsair or other RGB apps/programs.

**Comment by ZippyZolton** [comment]
I'm not exactly against this existing, but if the support is too "bloaty" it definitely won't be put into the main branch. I've also had exactly one game change my iCUE profile and either stopped playing it or immediately turned it off because my memory is really foggy on that. I don't remember if it screwed with my sound settings or macros, but the change probably bothered me.

**Reply by xTVaser** [reply]
iCUE support will never be added:
- https://www.corsair.com/us/en/s/icue-dev it requires applying to a developer program of which very very few games seem to have, in order to get access to their proprietary SDK and you basically need to be a commercial product.  It's also windows only
![image](https://github.com/open-goal/jak-project/assets/13153231/1c1ccb92-ad60-415b-affe-304e83fea0a3)

Razer's SDK is more sane https://developer.razer.com/works-with-chroma/download/ but I would be surprised if anyone spends the time to do it
